packages/pluggableWidgets/switch-web/src/Switch.tsx

```diff
@@ -0,0 +1,50 @@
+import { createElement, KeyboardEvent, MouseEvent, useCallback } from "react";
+import { executeAction, isAvailable } from "@mendix/piw-utils-internal";
+import SwitchComponent from "./components/Switch";
+
+import { SwitchContainerProps } from "../typings/SwitchProps";
+import "./ui/switch-main.scss";
+
+export default function Switch(props: SwitchContainerProps) {
```

Comment:
Any special reason to use default here? This component does not require default anymore and to also align with native we should use named export :)

This capability was added in [Mx9](https://docs.mendix.com/releasenotes/studio-pro/9.0) 
`We added support for named exported classes and functions in web widgets. You no longer need to export your widget class as a default.`

Reply:
No special reason. I thought to go for default since it's the only thing the module exports. I will change to named export.